### ISSUE TITLE
Color contrast issues

### ISSUE THREAD

**Issue by cncoulter** [cncoulter/Graph_Paper#9]
The color contrast between the blue link text and the white background has insufficient color contrast. Choose a darker shade of blue for link text. This is relevant to [WCAG 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).

**Comment by cncoulter** [comment]
Fixed!

https://github.com/cncoulter/Graph_Paper/commit/2a22a31cbc870db40cc57bf7dc97d411ca28a1d8
